### DISCUSSION TITLE
Importing TrekBuddy Format Maps

### DISCUSSION THREAD

**Discussion by phethuk** [peterLaurence/TrekMe#85] [Ideas]
With the demise of TrekBuddy, I wondered if any thought had been given to making use of TrekBuddy format maps/atlases in TrekMe? I thinking either as a direct import or via a conversion utility. I personnally have a number of atlases I have built up which would be a pain to recreate.

**Comment by p-lr** [comment]
A utility would be nice indeed. The TrekMe format is pretty simple and if you could create a converter I would add it to this repository (in the `scripts` folder). That could be useful for other people.
